src/neuroconv/tools/spikeinterface/spikeinterface.py

```diff
@@ -1563,6 +1568,33 @@ def write_waveforms(
         )
 
 
+def write_waveforms(
```

Comment:
Cool, two things and this is done:
1) Add one exactly like this for `add_waveforms`
2) We have a six months deprecation in neuroconv. Write in the NotImplementedError that the function will be completley removed at some point after February 2025

Reply:
done in last commit!